saleor/graphql/checkout/mutations.py

```diff
@@ -655,6 +655,13 @@ def perform_mutation(cls, _root, info, checkout_id, shipping_method_id):
 
 class CheckoutComplete(BaseMutation):
     order = graphene.Field(Order, description="Placed order.")
+    confirmation_needed = graphene.Boolean(
+        required=False,
```

Comment:
This is `required=False` on purpose? I mean are there cases where we want to return `null` here and not `true` or `false`?

Reply:
Well, good point.